### ISSUE TITLE
Datepicker: month dropdown list does not fully fit in screen for users with smaller or zoomed in browser page 

### ISSUE THREAD

**Issue by karolis-leonas** [sebgroup/green#1229]
### Bug already reported?

- [X] I confirm that I have checked if the bug already has been reported

### For which framework/library you are reporting the bug

Angular

### Component name

Datepicker/dropdown

### Description

Hello, we have noticed that in browser pages with smaller heights (not when page reaches mobile size, mind you) when Green datepicker is open, its month dropdown size does not fully fit within the whole page and some later months could stay hidden outside the page - it is not possible to scroll down to show later months because page itself is scrolled to the bottom in browser and list block overflows the bottom of the page.

We cannot override the datepicker month/year dropdown sizes on our project level because of the shadow-dom structure (or we might lack knowledge to do that, in which case please let us know), but I think it would be smarter if people who are responsible with Green could take a look at this and see if it could be fixed or not on a more global level since I believe other projects could also get hit by this issue.

We have noticed this behavior both before and after we did upgrades coming from https://github.com/sebgroup/green/pull/1219 pull request just so you know.

### Steps To Reproduce

We have noticed the issue at our project when running it as MFE within one of Business Arena's testing environments. There is a higher chance of replicating such issue if the selected account does not have transactions and user's browser page is of smaller height (either by using different browser size, smaller resolution or zooming in the browser itself).

Please contact me directly if you want to know the location of the project and more specific details on replication.

### Current Behaviour

Month dropdown list does not dynamically adjust to be smaller (in this case I use Edge browser on a screen with 1920x1200 resolution and 125% zoom in):
![image](https://github.com/sebgroup/green/assets/37910061/01f13739-2d64-4a28-853f-3749cf868735)

### Expected Behaviour

Month dropdown list height should either be smaller or automatically adjust with accordance on where datepicker and its month/year dropdowns are located in the page.
